packaging/package.sh

```diff
@@ -155,31 +170,30 @@ while [ $# -gt 0 ] ; do
                 usage
                 exit 1
             fi
-            shift
+            shift 2
             ;;
+
         -d | --distribution)
             TARGETDISTRO=$2
             if [ -z "$TARGETDISTRO" ] ; then
                 echo "Error: Missing target distribution"
                 usage
                 exit 1
             fi
-            shift
+            shift 2
             ;;
+
         -r | --release)
             RELEASE=$2
-            shift
+            shift 2
             ;;
-        -)
+
+        -*|*)
```

Comment:
@khos2ow this change, fails the packaging job when -o is passed.

Reply:
Correct. Shouldn't we update packaging job to have _correct_ flags rather than changing package.sh script to be able to tolerate _incorrect_ flags?

Reply:
The issue is to support the same jenkins jobs and scripts to work with packaging older releases. I think the script should ignore unknown flags and their values but not fail otherwise.